app/helpers/admin_helper.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
-module AdminHelper
+module AdminHelper # rubocop:disable Style/Documentation
+  def selectable_static_slots
+    StaticSlot.all.decorate.map { |static_slot| [static_slot.full_display, static_slot.id] }
+  end
+
+  def worked_hours_name_with_month(date)
```

Comment:
`def worked_hours_title_for_given_month(date)`

Put this in `AdminHelper::Members`